addons/point_of_sale/static/src/js/Screens/PaymentScreen/PaymentScreen.js

```diff
@@ -104,15 +104,12 @@ odoo.define('point_of_sale.PaymentScreen', function (require) {
             // click_tip
             const tip = this.currentOrder.get_tip();
             const change = this.currentOrder.get_change();
-            let value = tip.toFixed(this.env.pos.decimals);
-
-            if (tip === 0 && change > 0) {
-                value = change;
-            }
+            let value = tip === 0 && change > 0 ? change : tip;
```

Comment:
why removing the toFixed?

Reply:
explain in commit if needed :)

Reply:
> why removing the toFixed?

Because value is first a `string` then if there's no `tip` and there are some `change`, then `value` become a number lol
And it doesn't need to be a string in the first place